api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java

```diff
@@ -93,16 +93,13 @@ private void installConcepts() {
         	Context.flushSession(); //Flush so that purges are not deferred until after data import
         	
         	DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
-            dataImporter.importData("Reference_Application_Concepts-25.xml");
-            dataImporter.importData("Covid-19_Concepts-1.xml");
+            dataImporter.importData("Reference_Application_Concepts-24.xml");     
             
             if (ModuleUtil.compareVersion(OpenmrsConstants.OPENMRS_VERSION_SHORT, "2.2") < 0) {
-                dataImporter.importData("Reference_Application_Numeric_Concepts-25-pre2.x.xml");
-                dataImporter.importData("Covid-19_Numeric_Concepts-1_pre2.x.xml");
+                dataImporter.importData("Reference_Application_Numeric_Concepts-24-pre2.x.xml");
 			}
             else {
-                dataImporter.importData("Reference_Application_Numeric_Concepts-25-2.x.xml");
-                dataImporter.importData("Covid-19_Numeric_Concepts-1_2.x.xml");
+                dataImporter.importData("Reference_Application_Numeric_Concepts-24-2.x.xml");
```

Comment:
same here , do we have any package named `Reference_Application_Numeric_Concepts-24-2.x.xml`  ??

Reply:
check the resources folder for the right package names to import   https://github.com/openmrs/openmrs-module-referencemetadata/tree/master/api/src/main/resources

Reply:
@mozzy11 i have resolved that thanks